src/app/features/loader/loader.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { LoaderService } from '@core/services/loader/loader.service';
+
+@Component({
+  selector: 'app-loader',
+  templateUrl: './loader.component.html',
+  styleUrls: ['./loader.component.scss']
+})
+export class LoaderComponent implements OnInit {
+    loading: boolean;
+
+    constructor(
+        private loadingService: LoaderService
+    ) {
+        this.loadingService.isLoading.subscribe(res => {
```

Comment:
you could use this approach 
this.loading = this.loadingService.isLoading;
and implement `| async` in `if` condition